www/front_src/src/Resources/icons/Downtime.tsx

```diff
@@ -2,10 +2,8 @@ import React from 'react';
 
 import { SvgIcon, SvgIconProps } from '@material-ui/core';
 
-import { ReactComponent as IconDowntime } from './downtime.icon.svg';
-
 const Downtime = (props: SvgIconProps): JSX.Element => (
-  <SvgIcon component={IconDowntime} {...props} />
+  <div> to be loaded vite </div>
```

Comment:
Does the svg import fail with Vite ?

Reply:
yes, it fail to import the svg file directly inside typescript file

Reply:
fixed using vite-svgr